userapi/src/main/java/fk/prof/userapi/Cacheable.java

```diff
@@ -0,0 +1,14 @@
+package fk.prof.userapi;
```

Comment:
Should be part of cache package. While we are at this, why is cache nested inside api package.

Reply:
Moved to Cache Pacakge.